Spec/Auth.swift

```diff
@@ -1106,6 +1106,36 @@ class Auth : QuickSpec {
                         done()
                     }
                 }
+                
+                tokenParams.clientId = "newClientId"
+                tokenParams.ttl = 2000
+                tokenParams.capability = "{ \"test:*\":[\"test\"] }"
+                
+                waitUntil(timeout: testTimeout) { done in
+                    rest.auth.createTokenRequest(tokenParams, options: authOptions) { tokenRequest, error in
```

Comment:
> This API does not look right

Why do you think so? `createTokenRequest` API [has 3 arguments](https://github.com/ably/ably-ios/blob/master/Source/ARTAuth.h#L49-L50) - `ARTTokenParams` and `ARTAuthOptions` objects and a `callback` which provides `ARTTokenRequest` and an error if any.

> Why is this `options:` 

Lib written in `ObjC`, and tests in `Swift`, so `options:` is a part of `ObjC` method signature here and the argument is `authOptions` object of `ARTAuthOptions` class.